src/components/user-profile.component.js

```diff
@@ -104,99 +107,103 @@ export const UserProfile = ({
   navigation,
 }: Props) =>
   <View style={styles.container}>
-    <View style={styles.profile}>
-      <ImageZoom
-        uri={{
-          uri: initialUser.avatar_url
-            ? `${initialUser.avatar_url}&lastModified=${initialUser.updated_at}`
-            : `${user.avatar_url}&lastModified=${user.updated_at}`,
-        }}
-        style={[
-          styles.avatar,
-          (initialUser.type === 'User' || user.type === 'User') &&
-            styles.userAvatar,
-        ]}
-      />
-      <Text style={styles.title}>
-        {user.name || ' '}
-      </Text>
-      <Text style={styles.subtitle}>
-        {initialUser.login || ' '}
-      </Text>
-    </View>
-    <View style={styles.details}>
-      {user.hasOwnProperty('public_repos') &&
-      <TouchableOpacity
-        style={styles.unit}
-        onPress={() =>
-          navigation.navigate('RepositoryList', {
-            title: translate('user.repositoryList.title', language),
-            user,
-            repoCount: user.public_repos > 15 ? 15 : user.public_repos,
-          })}
-      >
-        <Text style={styles.unitNumber}>
-          {!isNaN(parseInt(user.public_repos, 10))
-            ? user.public_repos + (user.total_private_repos || 0)
-            : ' '}
-        </Text>
-        <Text style={styles.unitText}>
-          {translate('common.repositories', language)}
-        </Text>
-      </TouchableOpacity>}
-
-      {type !== 'org' && !isNaN(parseInt(starCount, 10)) &&
-        <TouchableOpacity style={styles.unit}>
-          <Text style={styles.unitNumber}>
-            {abbreviateNumber(starCount)}
+    {((user.hasOwnProperty('public_repos') &&
+      !isNaN(parseInt(starCount, 10))) ||
```

Comment:
Yeah we really want to be using lodash (or a simple utility library) to make this simpler. Please remind me to bring this up after our release